packages/core/ios/EmbraceManager.swift

```diff
@@ -577,14 +577,14 @@ class EmbraceManager: NSObject {
 
     private func errorCodeFrom(str: String) -> SpanErrorCode? {
         switch str {
-            case "Failure":
-                return .failure
-            case "UserAbandon":
-                return .userAbandon
-            case "Unknown":
-                return .unknown
-            default:
-                return nil
+        case "Failure":
```

Comment:
this was actually something a rule is asking for `switch_case_alignment`

it was throwing something like

```
packages/core/ios/EmbraceManager.swift:580:13: warning: Switch and Case Statement Alignment Violation: Case statements should vertically aligned with their closing brace (switch_case_alignment)
```